### PR TITLE
Add makeCartArc and tests

### LINKED ISSUE #140: v.1.4.0 [NewFunction] makeCartArc

### PR DESCRIPTION
**Pull request by waltsims**
closes #140

### CODE REVIEW

File: kwave/utils/mapgen.py [hunk 1]
```diff
@@ -2555,3 +2555,119 @@ def trim_cart_points(kgrid, points: np.ndarray):
     points = points[:, ind]
 
     return points
+
+
+import numpy as np
+import matplotlib.pyplot as plt
```
Comment (faridyagubbayli) [inline kwave/utils/mapgen.py:2561]: Should these be moved to the start of the file?

File: tests/matlab_test_data_collectors/matlab_collectors/collect_makeCartArc.m [hunk 2]
```diff
@@ -0,0 +1,22 @@
+% radius, num_points, center_pos
+all_params = {
+    {[0,0], 5.0, 7.0, [2,5], 50}, ...
+    {[2,2], 5.0, 7.0, [8,5], 5}, ...
+    {[2,9], 8.0, 13.0, [8,5], 5}
+    };  
+output_folder = 'collectedValues/makeCartArc/';
+
+for idx=1:length(all_params)
```
Comment (faridyagubbayli) [inline tests/matlab_test_data_collectors/matlab_collectors/collect_makeCartArc.m:9]: Why not to use the `TestRecorder`?

**Review by faridyagubbayli — APPROVED** [review]
Looks good but left two comments. Ideally, I'd like us to implement all new matlab function tests with `TestRecorder` but it is not a blocker.

File: tests/matlab_test_data_collectors/matlab_collectors/collect_makeCartArc.m (hunk 2 above)
Comment (waltsims) [reply tests/matlab_test_data_collectors/matlab_collectors/collect_makeCartArc.m:11]: done

File: kwave/utils/mapgen.py (hunk 1 above)
Comment (waltsims) [reply kwave/utils/mapgen.py:2561]: done

**Pull request merged**